src/commands/main.ts

```diff
@@ -239,6 +238,8 @@ export const createMainCommand = () => {
   createWatchCommand(program)
   createLogsCommand(program)
 
+  program.setAnalyticsPayload({ didEnableCompileCache })
```

Comment:
This will allow us to see the real-world perf impact

Reply:
Ideally, `module` would offer an API for checking if the cache is enabled, but oh well.

Given that this is diagnostic information about the environment, rather than threading it, I would break `maybeEnableCompileCache` into a module that captures the result of calling `module.enableCompileCache` and exports a function `isCompileCacheEnabled` that you can use here. That would let you avoid threading this information around and avoid this mysterious `boolean` argument.

If you continue to thread this information, I would move this into an options bag so it's more self-documenting (and to make room for other options later).

Reply:
It does provide that information. Good call. It wasn't even accurate to mark it as enabled in all cases. Done.